specification/2.0/Specification.adoc

```diff
@@ -572,20 +735,26 @@ The following example defines two buffer views: the first is an ELEMENT_ARRAY_BU
 }
 ----
 
-When a buffer view is used for vertex attribute data, it may have a `byteStride` property. This property defines the stride in bytes between each vertex.
+When a buffer view is used for vertex attribute data, it **MAY** have a `byteStride` property. This property defines the stride in bytes between each vertex. Buffer views with other types of data **MUST NOT** not define `byteStride`.
```

Comment:
Would it be valid for an extension to define a new "kind" of data that supports interleaving? Trying to accommodate `EXT_mesh_gpu_instancing` instance attributes and perhaps future extensions here.